### PR TITLE
Feature/add entities

### PR DESCRIPTION
**Pull request by DarPers**

### CODE REVIEW

File: ServiceDesk.DAL/Entities/User.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace ServiceDesk.DAL.Entities
+{
+    internal class User
+    {
+        public Guid Id { get; set; }
+
+        public string FirstName { get; set; } = string.Empty;
+
+        public string LastName { get; set; } = string.Empty;
+
```
Comment (VladMarshin) [inline ServiceDesk.DAL/Entities/User.cs:13]: Add here "name" and "description" fields also

File: ServiceDesk.DAL/Enums/Role.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace ServiceDesk.DAL.Enums
+{
+    internal enum Role
+    {
+        OperationAdministrator = 1,
```
Comment (VladMarshin) [inline ServiceDesk.DAL/Enums/Role.cs:5]: Add "None" here like in the next one

Comment (VladMarshin) [reply ServiceDesk.DAL/Enums/Role.cs:5]: And you can add SystemAdmin here

File: ServiceDesk.DAL/Entities/ExecutionRequest.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace ServiceDesk.DAL.Entities
+{
+    internal class ExecutionRequest
+    {
+        public Guid Id { get; set; }
+
+        public Guid TicketId { get; set; }
+
+        public Ticket Ticket { get; set; } = null!;
```
Comment (VadimZharikov) [inline ServiceDesk.DAL/Entities/ExecutionRequest.cs]: doubtful, but okay

File: ServiceDesk.DAL/Entities/Ticket.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public Guid Id { get; set; }
+
+    public string Name { get; set; } = string.Empty;
+
+    public string Description { get; set; } = string.Empty;
+
+    public Guid UserId { get; set; }
+
+    public User User { get; set; } = null!;
```
Comment (VadimZharikov) [inline ServiceDesk.DAL/Entities/Ticket.cs:15]: doubtful, but okay

**Review by VadimZharikov — APPROVED** [review]

**Pull request merged**